ras_party/controllers/account_controller.py

```diff
@@ -152,13 +153,10 @@ def change_respondent_enrolment_status(payload, session):
     session.commit()
 
     # If no enrolments are remaining for business/survey
-    # then send NO_ACTIVE_ENROLMENTS case event for each relevant B case
-    casegroup_ids = get_business_survey_casegroups(survey_id, business_id)
+    # then send NO_ACTIVE_ENROLMENTS case event
     enrolment_count = count_enrolment_by_survey_business(business_id, survey_id, session)
     if not enrolment_count:
-        logger.info('No active enrolments', business_id=business_id, survey_id=survey_id)
```

Comment:
keep this logging as a debug? also remove the doc string for the function its pointless

Reply:
or is that what the description does?